controllers/rcj_soccer_referee_supervisor/referee/supervisor.py

```diff
@@ -48,9 +52,17 @@ def __init__(self, match_time: int,
 
             self.robot_in_penalty_counter[robot] = 0
 
+            pc = ProgressChecker(progress_check_steps,
```

Comment:
```suggestion
            pc = ProgressChecker(
                progress_check_steps,
```

Reply:
Ok, this I really do not get. Can you please share a bit of reasoning behind this type of formatting?

Reply:
this is standard black formating style. But if you don't like it, let's stick with the current formating :)